index.js

```diff
@@ -34,4 +43,21 @@ async function fetchCharacters() {
   }
 }
 
+prevButton.addEventListener("click", () => {
+  if (page > 1) {
+    fetchCharacters(page - 1);
+    page--;
+    pagination.textContent = page + " / " + maxPage;
```

Comment:
You can just work with the global variable here. Just decrement the value of `page` and call `fetchCharacters`. As you already change the `textContent` of your pagination in your `fetchCharacters` function, you do not need to do this here again.
```suggestion
    page--;
    fetchCharacters();
```